tutorials/docs-10-using-turing-autodiff/index.qmd

```diff
@@ -62,4 +66,4 @@ c = sample(
 
 Generally, reverse-mode AD, for instance `ReverseDiff`, is faster when sampling from variables of high dimensionality (greater than 20), while forward-mode AD, for instance `ForwardDiff`, is more efficient for lower-dimension variables. This functionality allows those who are performance sensitive to fine tune their automatic differentiation for their specific models.
 
-If the differentiation method is not specified in this way, Turing will default to using whatever the global AD backend is. Currently, this defaults to `ForwardDiff`.
\ No newline at end of file
+If the differentiation method is not specified in this way, Turing will default to using whatever the global AD backend is. Currently, this defaults to `ForwardDiff`.

```

Comment:
When you fully switch to DI you can also use `DifferentiationInterfaceTest.benchmark_differentiation` for this ;)

Reply:
This is a good point. @torfjelde would it make more sense to just link to TuringBenchmarking.jl, rather than copying + pasting in a code snippet which may become outdated? Alternatively, is there a simple way to put example benchmarking code in here which runs whenever the docs are built?

Reply:
Thanks for reminding me about this @torfjelde . I've made the changes requested, so I'm going to mark this conversation closed. I'll get CI passing and then merge.